### PR TITLE
[istio] chore: configure to work on cert-manager v1

### PR DESCRIPTION
**Pull request by alejandroEsc**
**What type of PR is this?**

Chore

**What this PR does/ why we need it**:

Allow cert-manager v1 resources to be created

**Which issue(s) this PR fixes**:

https://jira.d2iq.com/browse/D2IQ-72170

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: staging/istio/charts/security/templates/certificate.yaml [hunk 1]
```diff
@@ -12,9 +16,14 @@ spec:
...
+{{- if  (.Capabilities.APIVersions.Has "cert-manager.io/v1") }}
+  subject:
+    organizations:
+      - D2iQ
+{{- else }}
   organization:
-  - D2iQ
-  secretName: istio-ca
+    - D2iQ
+{{- end }}  secretName: istio-ca
```
Comment (gracedo) [inline staging/istio/charts/security/templates/certificate.yaml:26]:
```suggestion
{{- end }}
  secretName: istio-ca
```

File: staging/istio/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: istio
-version: 1.6.9
+version: 1.6.10
```
Comment (shaneutt) [inline staging/istio/Chart.yaml:3]: I've been trying to keep the chart and app version aligned. How would you feel about folding in a patch of the app into this PR? yay/nay?

File: staging/istio/values.yaml [hunk 3]
```diff
@@ -13,7 +13,7 @@ istioOperator:
   profile: default
 
   hub: docker.io/istio
-  tag: 1.6.9
+  tag: 1.6.10
```
Comment (dkoshkin) [inline staging/istio/values.yaml:16]: Is this change intentional?

**Review by dkoshkin — APPROVED** [review]

**Review by gracedo — APPROVED** [review]

**Review by joejulian — APPROVED** [review]

File: staging/istio/values.yaml (hunk 3 above)
Comment (alejandroEsc) [reply staging/istio/values.yaml:16]: Yes, an ask by @shaneutt

Comment (shaneutt) [reply staging/istio/values.yaml:16]: tyvm :bow:
